### PR TITLE
Create a new pull-kubernetes-e2e-gce-bazel Jenkins job that uses Bazel for builds

### PR DESCRIPTION
**Pull request by ixdy**
This might work?
Might as well start experimenting, anyway.

/assign @fejta @spxtr

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -4065,6 +4065,20 @@
...
     ]
   }, 
+  "pull-kubernetes-e2e-gce-bazel": {
+    "args": [
+      "--env-file=platforms/gce.env", 
+      "--env-file=jobs/pull-kubernetes-e2e.env", 
+      "--env-file=jobs/pull-kubernetes-e2e-gce-bazel.env", 
+      "--build=bazel", 
+      "--stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-bazel", 
+      "--cluster="
```
Comment (krzyzacy) [inline jobs/config.json]: --mode=local

File: prow/config.yaml [hunk 2]
```diff
@@ -151,6 +151,11 @@ presubmits:
     rerun_command: "@k8s-bot pull-kubernetes-e2e-gce test this"
     trigger: "@k8s-bot pull-kubernetes-e2e-gce test this"
 
+  - name: pull-kubernetes-e2e-gce-bazel
+    context: pull-kubernetes-e2e-gce-bazel
+    rerun_command: "@k8s-bot pull-kubernetes-e2e-gce-bazel test this"
+    trigger: "@k8s-bot pull-kubernetes-e2e-gce-bazel test this"
```
Comment (krzyzacy) [inline prow/config.yaml]: assume you want to run it as a prow pod rather than a jenkins job? Then you'll need to fill up the pod spec here.

Comment (ixdy) [inline prow/config.yaml]: I meant Jenkins job for first pass. Forgot I had to add those configs too, but done now.

**Comment by krzyzacy** [comment]
/lgtm

sleeping travis :-(

**Comment by ixdy** [comment]
Gonna merge so I can iterate. Long-term this job will eventually be deleted.

**Pull request merged**